zuul.d/layout.yaml

```diff
@@ -0,0 +1,70 @@
+---
+# zuul.d/layout.yaml
+- job:
+    name: ansible-tox-devel
```

Comment:
lets scope this to molecule-tox-devel, since job names have to be unique.  This will help somebody know they are 'molecule' based, where ansible is more toplevel